### ISSUE TITLE
Remove From Account Wording

### ISSUE THREAD

**Issue by Protektor-Desura** [desura/Desurium#340]
When you right click on a game in the game list and then select uninstall you are presented with two checkboxes. One for remove files and one for "Remove from Account". 

That is poor wording for what is really happening. What is really happening is you are hiding the game from the game list so it doesn't show up anymore. It is not removed from your account and you still own the game. 

The checkboxes should be removed and have two options in the rightclick menu, Uninstall and Hide. It isn't clear that you need to uninstall a game just to hide a game in the list. 

Comments?

**Comment by LiamDawe** [comment]
Agreed horrible wording currently. Needs to be made much clearer.

**Comment by karolherbst** [comment]
Because this is a little change I'm thinking about adding it to the 0.8 milestone

**Comment by LiamDawe** [comment]
I would agree on that wouldnt it be a case of just editing the language strings?

**Comment by karolherbst** [comment]
no, because of "The checkboxes should be removed and have two options in the rightclick menu, Uninstall and Hide. It isn't clear that you need to uninstall a game just to hide a game in the list."

but we can rename them in the first run

**Comment by Protektor-Desura** [comment]
The logic can be called with "Uninstall" being done as if only "Remove Files" box was checked. "Hide" can be as if both of the current boxes were selected/checked, "Remove Files" and "Remove from Account". If you currently check both boxes when a game is not installed it doesn't have a problem with it and just hides the game from the list.
